app/assets/stylesheets/modern-business.scss

```diff
@@ -241,3 +247,33 @@ footer {
   font-size: 40px;
   text-align: center;
 }
+
+/* Countdown Clock */
+
+#clockdiv{
+  font-family: sans-serif;
+  color: #fff;
+  display: inline-block;
+  font-weight: 50;
+  text-align: center;
+  font-size: 30px;
+}
+
+#clockdiv > div{
+  padding: 10px;
+  border-radius: 2px;
+  background: #044A8E;
+  display: inline-block;
+}
+
+#clockdiv div > span{
```

Comment:
Avoid using id selectors<br>Merge rule `#clockdiv div > span` with rule on line 253<br>Selector should have depth of applicability no greater than 2, but was 3<br>Opening curly brace `{` should be preceded by one space